modules/ding_frontend/ding_frontend.info

```diff
@@ -137,5 +133,8 @@ features[views_view][] = ding_multiple_search
 features[views_view][] = ding_node_search
 features[views_view][] = ding_related_content
 features_exclude[menu_blocks][] = 2
-features_exclude[menu_blocks][] = 2
```

Comment:
How is this related to the case?

Reply:
In latest development branch, structure of changed file matches with default state.